src/node_api.cc

```diff
@@ -156,14 +156,20 @@ class HandleScopeWrapper {
 // across different versions.
 class EscapableHandleScopeWrapper {
  public:
-  explicit EscapableHandleScopeWrapper(v8::Isolate* isolate) : scope(isolate) {}
+  explicit EscapableHandleScopeWrapper(v8::Isolate* isolate) :
+    scope(isolate), escape_called_(false) {}
```

Comment:
Minuscule style nit: the colon should go on the next line and have 4 spaces of indent.